tests/dummy/app/view-components/nrg-search/template.hbs

```diff
@@ -4,22 +4,21 @@
   <!-- END-SNIPPET  -->
 </NrgCodeBlock>
 
-<NrgCodeBlock @label="Loading" @codeBlockName="loading-search.hbs">
+<NrgCodeBlock @label="Querying" @codeBlockName="querying-search.hbs">
   <p>
-    Searching for <strong>{{or loadingQuery "Nothing"}}</strong>
-  </p>
-  <!-- BEGIN-SNIPPET loading-search -->
-  <NrgSearch @loading={{true}} @query={{action (mut loadingQuery)}} />
-  <!-- END-SNIPPET  -->
-</NrgCodeBlock>
-
-<NrgCodeBlock @label="Querying Template" @codeBlockName="querying-search.hbs">
-  <p>
-    Selected <strong>{{or selected.header "Nothing"}}</strong>
+    Searching for
+    <strong>{{or this.queryString "Nothing"}}</strong>
```

Comment:
I believe you only need to prefix a property with `this` when it's inside a component; data in routes can only come from the route itself.